### DISCUSSION TITLE
Unclear error message

### DISCUSSION THREAD

**Discussion by cova-fe** [folke/noice.nvim#845] [Q&A] [answered]
Hi all,
I installed noice plugin but now I'm getting this error message:
![Screenshot 2024-06-09 at 12 44 05](https://github.com/folke/noice.nvim/assets/385249/9f58a3b8-e59c-4ed4-914a-295187dc47d4)
Now, I can't understand if it is a bug, a misconfiguration on my side or something else, however the fact that noice reports "nvim" as plugin that is interfering with his setup looks a bit weird. 
Checkhealth output:
![Screenshot 2024-06-09 at 18 42 17](https://github.com/folke/noice.nvim/assets/385249/3996ab16-3a26-4bd2-8a5e-39193fcbbdfc)
Can someone clarify what is going on here?
many thanks!

**Comment by ilias777** [comment, marked as answer]
I think it's overwritten by [cmp-nvim-lsp-signature-help](https://github.com/hrsh7th/cmp-nvim-lsp-signature-help)

Try to remove `cmp-nvim-lsp-signature-help` from `cmp`.

**Comment by cova-fe** [comment]
Thanks for the hint, it seems I found the culprit: in a remote corner of the configuration I had _ray-x/lsp_signature.nvim_ still active. I removed it and apparently the issue disappeared. 
Thanks again, it would have took ages for me to find it.
Pity that the warning message from noice is not spot on, but I can live with that :)
